packages/website/guides/build-with-cannon.mdx

```diff
@@ -24,195 +26,218 @@ after:
-  </AlertDescription>
-</Alert>
+### Step 1: Build your Cannon package
+
+To start, let's build the Cannon package using the `cannon build` command. This command uses your `cannonfile.toml` to prepare an local image for use on a local node for development, or as a blueprint for a deployment on live networks.
+
+1. Open your terminal.
+2. Run the following command:
+
+   ```bash
+   cannon build
+   ```
+
+   This command uses your `cannonfile.toml` by default. You can also specify a different path to any Cannonfile. Learn more about the `build` command in the [docs](/learn/cli#build).
+
+   The resulting CLI output will look like this:
+
+   ```bash
+   Building the foundry project...
+   forge build succeeded
+
+   Anvil instance running on: http://127.0.0.1:59576
+
+   Checking for existing package...
+   Starting fresh build...
+
+   Initializing new package...
+   Name: learn-cannon
+   Version: 0.0.1
+   Preset: main (default)
+   Chain ID: 13370
+
+   Building the chain (ID 13370) via http://127.0.0.1:59576/...
+
+   Executing [deploy.Counter]...
+     ✔ Successfully deployed Counter
+     Contract Address: 0xACEbBC3E0D8BC1bB13a35b40f3714A7c78C158f7
+     Transaction Hash: 0x0abfae8229490349f80230414259ca1fcc60eecead61212733154fb3c80feae1
+     Transaction Cost: 0.000213466 ETH (106,733 gas)
+
+   Executing [var.ProtocolSettings]...
+     Setting: number = 420
+
+   Executing [invoke.SetNumber]...
+     ✔ Successfully called setNumber(420)
+     Signer: 0xEB045D78d273107348b0300c01D29B7552D622ab
+     Contract Address: 0xACEbBC3E0D8BC1bB13a35b40f3714A7c78C158f7
+     Transaction Hash: 0x4ceea742fecb9ba8adce1d364beb01f17dc93125b740e532bbdd8ff1068b96eb
+     Transaction Cost: 0.00007268124780963 ETH (43,506 gas)
+
+   Writing upgrade info...
+   💥 learn-cannon:0.0.1@main built for Cannon (Chain ID: 13370)
+   This package can be run locally and cloned in cannonfiles.
+
+   Package data has been stored locally
+   ╔═════════════════╤═══════════════════════════════════════════════════════╗
+   ║ Deployment Data │ ipfs://QmVpLGJMKqAd5hD3LgM3oh37NfbmzTS9FUyjCZx1LcB8yW ║
+   ╟─────────────────┼───────────────────────────────────────────────────────╢
+   ║ Package Code    │ ipfs://QmVhGysWPz2toaL9FVWoyTTgp2Atf4UMESz4Dj5dKvRhWb ║
+   ╟─────────────────┼───────────────────────────────────────────────────────╢
+   ║ Metadata        │ ipfs://QmRsYpSHrvjVmktXJtLYZbmiHD8GNKp64qxoZ2hbGcqEDf ║
+   ╚═════════════════╧═══════════════════════════════════════════════════════╝
+
+   Publish learn-cannon:0.0.1 to the registry and pin the IPFS data to
+   > cannon publish learn-cannon:0.0.1 --chain-id 13370
+
+   Run this package
+   > cannon learn-cannon:0.0.1
+   ```
+
+   <Alert variant="info">
+     <AlertDescription>
+       If you'd like to keep the node running after the build process, add the `--keep-alive` flag to the build command. For example: `cannon build --keep-alive`.
+     </AlertDescription>
+   </Alert>
```

Comment:
IMHO, I would remove this alert here and just leave it in the API docs. As this guide is intended for newish users and the --keep-alive is an super specific case, and users normally don't know whats the "build node" to keep running, etc

```suggestion
```

Reply:
sorry missed this here, can do it later